test/hooks/test-trace-grpc.js

```diff
@@ -16,7 +16,8 @@
 'use strict';
 
 var common = require('./common.js');
-require('../..').private_().config_.enhancedDatabaseReporting = true;
+var trace = require('../..')();
+trace.startAgent().private_().config_.enhancedDatabaseReporting = true;
```

Comment:
Can this be passed through the config object to `startAgent`? I know this was mirroring previous style but passing it the official way would be better.